x-pack/plugins/apm/server/routes/service_map/get_service_map_service_node_info.ts

```diff
@@ -358,19 +357,28 @@ function getMemoryStats({
     };
 
     let memoryUsage = await getMemoryUsage({
+      script: percentCgroupMemoryUsedScript,
       additionalFilters: [
-        { exists: { field: METRIC_CGROUP_MEMORY_USAGE_BYTES } },
+        {
+          bool: {
+            should: [cgroupMemoryFilter],
+            minimum_should_match: 1,
+          },
+        },
```

Comment:
What about extracting everything, including the `should` clause?
```suggestion
        cgroupMemoryFilter
```

Reply:
nice suggestion fixed it https://github.com/elastic/kibana/pull/145348/commits/385f9b7dca6db2168187cd3d48004ed90e89e0eb